src/nccl_ofi_net.c

```diff
@@ -424,18 +429,25 @@ int nccl_net_ofi_info_properties(struct fi_info *nic_prov, int dev_id, int num_d
 		 * this is probably ok; any affinity is lost by
 		 * bouncing through host buffers anyway.
 		 */
-		if (active_cuda_device / gpus_per_conn  != dev_id) {
-			for (c=strlen(dev_props.pci_path); c && dev_props.pci_path[c] != '/'; c--) {
-				dev_props.pci_path[c] = '\0';
+		if ((active_cuda_device / gpus_per_conn != dev_id) && props->pci_path) {
```

Comment:
Why might `props->pci_path` be NULL here? Won't we just crash when we try to `NCCL_OFI_TRACE` it below?

Seems like we should handle that case separately and just return an error?